examples/pull-progress/pull.ts

```diff
@@ -0,0 +1,25 @@
+import ollama from 'ollama'
+
+const model = 'falcon'
```

Comment:
let's switch this to `mistral`

```suggestion
const model = 'mistral'
```